File: src/main/java/factorio/debugger/DAP/messages/requests/DAPThreadsRequest.java
package factorio.debugger.DAP.messages.requests;

import com.fasterxml.jackson.annotation.JsonTypeName;
import factorio.debugger.DAP.messages.DAPAdditionalProperties;
import factorio.debugger.DAP.messages.DAPRequest;

@JsonTypeName("threads")
public class DAPThreadsRequest extends DAPRequest<DAPThreadsRequest.ThreadsRequestArguments> {
    public static class ThreadsRequestArguments extends DAPAdditionalProperties {

    }
}

File: src/main/kotlin/factorio/debugger/DAP/messages/requests/DAPThreadsRequest.kt
package factorio.debugger.DAP.messages.requests

import com.fasterxml.jackson.annotation.JsonTypeName
import factorio.debugger.DAP.messages.DAPAdditionalProperties
import factorio.debugger.DAP.messages.requests.DAPThreadsRequest.ThreadsRequestArguments

@JsonTypeName("threads")
class DAPThreadsRequest : DAPRequest<ThreadsRequestArguments>(ThreadsRequestArguments()) {
    class ThreadsRequestArguments : DAPAdditionalProperties()
}
